### ISSUE TITLE
[UPG FAIL] After stage 4 Grub Failed to load

### ISSUE THREAD

**Issue by greenot** [cpanel/elevate#324]
I had console access and stage four appeared to go fine, all 1000+ packages got installed.  It rebooted at the end of stage four but it ended up in a grub shell.

The system was a UEFI Boot with software raid on the system drives.

The server did not have anything critical on it so we wiped it and just installed Alma 8 fresh, this was mostly a test.

**Comment by toddr** [comment]
If you open a ticket with support, we can take a look. Can you tell me if these kernels have additional kernel arguments? Do they boot via grub2? Does the kernel need external drivers for the RAID? what package do they come from and do they work on 7 and 8?

**Comment by toddr** [comment]
Any new migrations?

**Comment by greenot** [comment]
No, not yet, not with that particular config.  Once I get a server 
cancellation on one of these I plan to attempt it again to see in a 
situation where there woul;d be no harm should there be a failure.



Jon Berry
President
Green Olive Tree
(757) 397-9785 <tel:(757) 397-9785>
***@***.***
Review us on Google <https://g.page/r/Ce9Z-yOCWGbSEB0/review>
<https://go.scheduleyou.in/znZNUY22T?cid=is:~Contact.Id~>


------ Original Message ------
From "Todd Rinaldo" ***@***.***>
To "cpanel/elevate" ***@***.***>
Cc "greenot" ***@***.***>; "Mention" 
***@***.***>
Date 4/23/2024 7:38:37 PM
Subject Re: [cpanel/elevate] [UPG FAIL] After stage 4 Grub Failed to 
load (Issue #324)

**Comment by rch77** [comment]
Is the Elevate tested at all with UEFI boot from md* software raid? Anybody tried it? I.e. /boot /boot/efi / partitions on regular md raid1 on centos 7.

**Comment by greenot** [comment]
I still have not retested it. I have a client asking for it but they are in this condition set so I begged off for now. Jon BerryGreen Olive Tree Inchttps://greenolivetree.net(757) 397-9785Skype: GreenOTOn Jun 14, 2024, at 6:03 PM, rch77 ***@***.***> wrote:﻿
Is the Elevate tested at all with UEFI boot from md* software raid? Anybody tried it? I.e. /boot /boot/efi / partitions on regular md raid1 on centos 7.
